x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -122,15 +137,57 @@ const EventDetailsComponent: React.FC<Props> = ({
     [alerts, browserFields, data, id, isAlert, timelineId, timelineTabType]
   );
 
-  const selectedTab = useMemo(() => tabs.find((t) => t.id === view) ?? tabs[0], [tabs, view]);
+  const selectedEventTab = useMemo(() => tabs.find((t) => t.id === eventView) ?? tabs[0], [
+    tabs,
+    eventView,
+  ]);
+
+  const isThreatPresent: boolean = useMemo(
+    () =>
+      selectedEventTab.id === tabs[0].id &&
+      isAlert &&
+      !!data.find((item) => item.field === 'threat.indicator'),
+    [tabs, selectedEventTab, isAlert, data]
+  );
+
+  const threatTabs: EuiTabbedContentTab[] = useMemo(() => {
+    const defaultProps = { data, eventId: id, timelineId };
+    return isAlert && isThreatPresent
+      ? [
+          getSummaryTab(EventsViewType.threatSummaryView, i18n.THREAT_SUMMARY, {
+            ...defaultProps,
+            isDisplayingThreatSummary: true,
+          }),
+          getSummaryTab(EventsViewType.threatInfoView, i18n.THREAT_INFO, {
+            ...defaultProps,
+            isDisplayingThreatInfo: true,
```

Comment:
<kbd>
<img width="1272" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/112908749-b5e38800-90b5-11eb-8887-8be6d28f5b2a.png">
</kbd>


UX-wise, there is no indication that there is more data to scroll within the "Threat Info" tab until I actually try to scroll it (and even then, sometimes the Alert Details panel captures the scroll event instead. I don't have a solution to offer, but I wanted to call this out as a potential concern.

Reply:
added some css to ensure that the scrollbar is always visible if there is y overflow, which made it better. as for the scroll capture, mouse has to be exactly within the indicated area (and not on the padding of the parent etc.), please let me know if any further improvements should be made.

<img width="832" alt="Screen Shot 2021-03-31 at 1 26 50 PM" src="https://user-images.githubusercontent.com/18617331/113185745-d5e58980-9224-11eb-977e-506e7f96c7bd.png">